src/applications/travel-pay/tests/containers/TravelPayStatusApp.unit.spec.jsx

```diff
@@ -226,6 +226,29 @@ describe('App', () => {
     });
   });
 
+  it('displayed status filters correctly', async () => {
```

Comment:
We need to make sure we include tests that cover [the acceptance criteria outlined in the ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/83451).